### PR TITLE
add various grammar and copy improvements

### PR DESCRIPTION
**Pull request by jipiboily**

### CODE REVIEW

File: docs/installation/docker_image.rst [hunk 1]
```diff
@@ -10,7 +10,7 @@ In order to be able to download the Gemnasium Enterprise docker image from the `
 authenticated on docker hub, and provide the username used to pull the image to
 Gemnasium.
 
-.. todo: We'll probably leave the gemnasium/enterprise image public, to simplify the installation
+To have access to the Gemnasium Enterprise image, you need to have a license from us, and we will then give you access to it via the Docker Hub. It is a private image.
```
Comment (gravis) [inline docs/installation/docker_image.rst]: Not really accurate: actually, we will authorize clients to access to our docker image, but they need to provide a  docker user for that. The licence sent will permit to run and sync with our servers.

File: docs/installation/firewall.rst [hunk 2]
```diff
@@ -43,7 +43,7 @@ In return, Gemnasium syncer provides all the metadata corresponding to this requ
 
 .. note:: Private dependencies are completely ignored by the syncer.
 
-Advisories can be created at any time on Gemnasium.com, that's why your instance synchronizes every five minutes.
-If one of your project is using a dependency affected by a security issue, you will be instantly notified by your instance.
+Advisories can be created at any time on Gemnasium.com; that's why your instance synchronizes every five minutes.
```
Comment (gravis) [inline docs/installation/firewall.rst]: I think this is not the case anymore, it's hourly now (I know it's not one of your change, but let's keep the doc clean :)

**Review by gravis — CHANGES_REQUESTED** [review]

File: docs/installation/docker_image.rst (hunk 1 above)
Comment (jipiboily) [reply docs/installation/docker_image.rst]: That's what I meant, will rephrase.

File: docs/installation/firewall.rst (hunk 2 above)
Comment (jipiboily) [reply docs/installation/firewall.rst]: You think...can you double check please? :)

Comment (gravis) [reply docs/installation/firewall.rst]: `"@hourly"` in the code, confirmed :)

File: docs/installation/docker_image.rst (hunk 1 above)
Comment (jipiboily) [reply docs/installation/docker_image.rst]: > To have access to the Gemnasium Enterprise image, send us the Docker Hub user you want; we will share the image with that user.

?

File: docs/installation/firewall.rst (hunk 2 above)
Comment (jipiboily) [reply docs/installation/firewall.rst]: Fixed.

File: docs/installation/docker_image.rst (hunk 1 above)
Comment (jipiboily) [reply docs/installation/docker_image.rst]: I pushed that version.

**Pull request merged**